eZ/Bundle/EzPublishCoreBundle/EventListener/ViewControllerListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 
     public static function getSubscribedEvents()
     {
-        return array( KernelEvents::CONTROLLER => 'getController' );
+        return [KernelEvents::CONTROLLER => ['getController', 10]];
```

Comment:
Maybe fix this in your commit so it uses standard array syntax? No need to change *that* here.